### ISSUE TITLE
>40 Characters Comment Requirement a good thing? 

### ISSUE THREAD

**Issue by Raskaro** [safing/stamp.community#70]
Is the 40 Character Requirement a good Feature? I just tried to enter "Google Ad Plattform" for DoubleClick and thought thats enough info. Maybe add some Text/Tooltip what could/should be mentioned in the Comment ...

**Comment by dhaavi** [comment]
Good idea, until then you can just be creative: https://stamp.community/label_stamps/22
#notHelpful
